sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -998,21 +998,6 @@ describe("Errors after close()", function(): void {
   describe("Errors after close() on receiver", function(): void {
     const entityToClose = "receiver";
 
-    it("Unpartitioned Queue: errors after close() on receiver #RunInBrowser", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.UnpartitionedQueue,
-        TestClientType.UnpartitionedQueue,
-        entityToClose
-      );
-
-      await testReceiver(
-        getReceiverClosedErrorMsg(receiverClient.entityPath, ClientType.QueueClient, false)
-      );
-      await testAllDispositions();
-    });
-
```

Comment:
Why is this change in this PR?

Reply:
This test expects that the dispositions fail if the receiver is closed. Removed the test for non-session entity since that's not the case anymore.